doc/guides/contributing/pull-requests.md

```diff
@@ -115,13 +115,13 @@ If you are modifying code, please be sure to run `make lint` from time to
 time to ensure that the changes follow the Node.js code style guide.
 
 Any documentation you write (including code comments and API documentation)
-should follow the [Style Guide](../../STYLE_GUIDE.md). Code samples included
-in the API docs will also be checked when running `make lint` (or
+should follow the [Style Guide](../documentation-style.md). Code samples
```

Comment:
Either this should be `doc-style-guide` (like `cpp-style-guide`), or `cpp-style-guide` should be `cpp-style` (like this).

Reply:
+1 to `doc-style-guide.md`